src/main/java/oc/wh40k/units/im/IMWolfGuardTerminators.java

```diff
@@ -0,0 +1,87 @@
+package oc.wh40k.units.im;
+
+import oc.*;
+
+public class IMWolfGuardTerminators extends Eintrag {
+
+    AnzahlPanel squad;
+    OptionsZaehlerGruppe o1, o1x;
+    OptionsZaehlerGruppe o2, o2x;
+    OptionsZaehlerGruppe o4;
+    OptionsZaehlerGruppe wolfsklauen;
+    RuestkammerStarter rkBoss3;
+
+
+    public IMWolfGuardTerminators() {
+        name = "Wolf Guard Terminators\n";
```

Comment:
Newlines are in those names because of a bad interaction between unit names and AnzahlPanels. You won't need a name field, if both names are identical. Which they weren't in the past and currently most likely are. I'll remove those name fields where possible.